### PR TITLE
finished code for startup-unicorn

### PR DESCRIPTION
**Pull request by bavaria95**

### CODE REVIEW

**Comment by bavaria95** [comment]
Some discussion
blahblah

File: t.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "41"
```
Comment (dimkaboyko) [inline t.rb]: this code smells

**Comment by bavaria95** [comment]
one more comment

File: t.rb (hunk 1 above)
Comment (dimkaboyko) [inline t.rb]: it should have been "42" :(

Comment (bavaria95) [inline t.rb]: @dimkaboyko I know your perfectionism, so it's ok ;)

**Pull request merged**